spec/services/search_spec.rb

```diff
@@ -67,11 +66,9 @@
                                  linkage_group_label: 'linkage group label')
     lg2 = create(:linkage_group, consensus_group_assignment: 'no consensus',
                                  linkage_group_label: 'group2')
-    lm1 = create(:linkage_map, linkage_map_label: 'linkage map label',
-                               map_version_no: '1',
+    lm1 = create(:linkage_map, map_version_no: '1',
                                plant_population: pp1)
-    lm2 = create(:linkage_map, linkage_map_label: 'linkage map label',
-                               map_version_no: '333',
+    lm2 = create(:linkage_map, map_version_no: '333',
                                plant_population: pp1)
     mp1 = create(:map_position, map_position: '102.8',
                                 linkage_group: lg1,
```

Comment:
There is something odd, then. You see there are 2-3 objects per each model and they usually have non-unique attributes assigned to them (that is actually kind of point here, as we want to test searching).

Anyway, removing parameters as you did here is not a good solution for that test. If it keeps giving you failures after the factory changes, I'll try to look into it.